selfdrive/car/honda/interface.py

```diff
@@ -198,6 +198,17 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       tire_stiffness_factor = 0.75
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.2], [0.05]]
 
+    elif candidate == CAR.FREED:
```

Comment:
merge this with the fit parameters, i.e. `elif candidate in [CAR.FIT, CAR.FREED]`

Reply:
Oh, actually I thought I had the proper mass and wheelbase here.  I'll correct those, and this entry will be different than FIT.  (Freed is a minivan, Fit is a compact.)